packages/mobile/src/screens/settings-screen/EmailFrequencyControlRow.tsx

```diff
@@ -0,0 +1,54 @@
+import { useCallback } from 'react'
+
+import { updateEmailFrequency } from 'audius-client/src/common/store/pages/settings/actions'
+import { getEmailFrequency } from 'audius-client/src/common/store/pages/settings/selectors'
+import { EmailFrequency } from 'audius-client/src/common/store/pages/settings/types'
+
+import { SegmentedControl } from 'app/components/core'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+
+import { SettingsRowLabel } from './SettingRowLabel'
+import { SettingsRow } from './SettingsRow'
+import { SettingsRowContent } from './SettingsRowContent'
+
+const messages = {
+  emailFrequency: '‘What You Missed’ Email Frequency',
```

Comment:
Should this be back tick on the outside and regular single quote on the inside?

Reply:
yeah exactly, good call